### PR TITLE
Remove carthage folder

### PR DESCRIPTION
**Pull request by AYastrebov**
- Remove carthage folder from repo (the same as https://github.com/pinterest/PINCache/pull/306)
- Use https://github.com/SDWebImage/libwebp-Xcode as webp dependency

### CODE REVIEW

File: Cartfile [hunk 1]
```diff
@@ -1,3 +1,3 @@
-git "https://chromium.googlesource.com/webm/libwebp" "v1.1.0"
```
Comment (garrettmoon) [inline Cartfile:1]: Can you put this in a separate PR?

**Comment by garrettmoon** [comment]
Like the PINCache PR I don't think we can just remove Carthage – the current setup has the project built with it.

File: Cartfile (hunk 1 above)
Comment (AYastrebov) [reply Cartfile:1]: Sure, will make it tomorrow

**Comment by AYastrebov** [comment]
> Like the PINCache PR I don't think we can just remove Carthage – the current setup has the project built with it.

Ok, I'll take a closer look at that.

**Comment by AYastrebov** [comment]
@garrettmoon this would be also nice to check on CI. Thanks!
